.github/workflows/cd.yml

```diff
@@ -0,0 +1,60 @@
+name: CD
```

Comment:
I think we need a doc in `docs/cd_with_aws.md` or similar so we can clarify the pre-requisites (AWS account, ECR set up, vars and secrets in the repo, etc).

In the future we might have more like `docs/cd_with_gh.md` and `docs/cd_with_heroku.md`

Reply:
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

Reply:
https://devcenter.heroku.com/articles/container-registry-and-runtime#building-and-pushing-image-s